src/devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -566,6 +566,7 @@ bool WholeBodyDynamicsDevice::loadSettingsFromConfig(os::Searchable& config)
     settings.forceTorqueFilterCutoffInHz = 3.0;
     settings.jointVelFilterCutoffInHz    = 3.0;
     settings.jointAccFilterCutoffInHz    = 3.0;
+    useSkinContact                       =false;
```

Comment:
I suggest to remove this parameter, so the skin position readings works out of the box if you just connect the ports, as it happened on the older version of wholeBodyDynamics .